x-pack/plugins/watcher/server/models/action_status/__tests__/action_status.js

```diff
@@ -107,8 +108,10 @@ describe('action_status', () => {
       });
 
       describe(`correctly calculates ACTION_STATES.ERROR`, () => {
-        it('lastExecutionSuccessful is equal to false', () => {
+        it('lastExecutionSuccessful is equal to false and it is the most recent execution', () => {
           upstreamJson.actionStatusJson.last_execution.successful = false;
+          upstreamJson.actionStatusJson.last_execution.timestamp =
```

Comment:
I'm not 100% sure what this assignment is doing?

Reply:
Not sure what I was going for there either 😂 . Removed.